build-system/tasks/clean.js

```diff
@@ -30,7 +35,9 @@ async function clean() {
     'build-system/server/new-server/transforms/dist',
     'build-system/tasks/performance/cache',
     'build-system/tasks/performance/results.json',
-    'build-system/global-configs/custom-config.json',
+    // Custom configs are not deleted by default, see flag --include_custom_configs
+    // 'build-system/global-configs/custom-config.json',
+    // 'build-system/global-configs/custom-flavors-config.json',
```

Comment:
Suggestion applied in de913d298b (thought I do worry a bit that these paths will accidentally be added when someone follows the instructions at the top of the file list:
```
// Keep this list in sync with .gitignore, .eslintignore, and .prettierignore
```

Reply:
Aha, I see your point. Since the config json files don't really belong in the "Output directories" section, perhaps we can create a new section called "Custom config files" in the three dotfiles, and add the comment about keeping them in sync at line 51?

Reply:
Good idea. I can do this small shuffle in a separate PR.